### ISSUE TITLE
[Feature Request] Setting to skip database selection and go to last recently used

### ISSUE THREAD

**Issue by GeorgePetri** [PhilippC/keepass2android#278]
I use only one database so once I play open the app, either through the icon or autofill, I have to select every time the same database. 
Having a setting to get to the enter password screen on the last used database when entering the app would simplify the enter password flow.

Thanks!

**Comment by mcarver2000** [comment]
Are you using Google Drive? There appears to be issues with how Google allows 3rd party apps to automatically grab a file. Other cloud and local files don't have this issue.

**Comment by GeorgePetri** [comment]
Yes I use Google Drive. Is there a specific Google Drive issue that makes automatically opening a file impossible?

**Comment by mcarver2000** [comment]
I believe the Google Drive issue is currently being addressed in the current betas.

**Comment by GeorgePetri** [comment]
That's a shame, hopefully it will get fixed.

**Comment by jkaberg** [comment]
Dupe of https://github.com/PhilippC/keepass2android/issues/216
